docs/operate/get-started/setup.md

```diff
@@ -31,33 +25,30 @@ aliases:
   - /get-started/installation/
 ---
 
-Get started by installing the open-source software that drives your hardware and connects your device to the cloud.
-The easiest way to do this is through the Viam app, so that your machines are automatically connected to configuration and remote operation tools.
+Get started by installing [`viam-server`](/operate/reference/viam-server/),the open-source software that drives your hardware and connects your device to the cloud.
 
 {{< expand "Supported systems" >}}
 
-Viam can run on any computer that runs one of the following operating systems:
+`viam-server` can run on any computer that runs one of the following operating systems:
 
 - Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
 - macOS
 - Windows
-  - native: use native if you are using a WSL version prior to WSL 2 or need native USB support
-  - WSL: use WSL if you are using python modules or other linux dependencies
+  - Native: Use native if you are using a WSL version prior to WSL 2 or need native USB support
+  - WSL: Use WSL if you are using Python modules or other Linux dependencies
 
-Viam also offers a lightweight binary to support the following 32-bit microcontrollers:
-
-- [ESP32-WROVER Series](https://www.espressif.com/en/products/modules/esp32)
-
-ESP32 microcontrollers must have at least 2 cores, 384kB SRAM, 2MB PSRAM and 8MB flash to work with Viam.
-
-Viam can run on Windows Subsystem for Linux (WSL), but WSL itself does not currently support exposing many types of Windows hardware to the embedded Linux kernel.
+`viam-server` can run on Windows Subsystem for Linux (WSL), but WSL itself does not currently support exposing many types of Windows hardware to the embedded Linux kernel.
 This means that some hardware, such as a connected webcam, may not be available to `viam-server` with WSL, even though it is fully supported for native Linux systems.
 
+For 32-bit systems, see [Set up an ESP32](/operate/get-started/setup-micro/).
+
 {{< /expand >}}
```

Comment:
This seems very similar to the "What platforms does Viam run on?" section in "Learn the basics."

I think we should just have the content in one of the two places.

If it makes more sense here, I think we could move this out of the dropdown, call it "Supported operating systems", and move the carousel here. That would also mirror the "Supported microcontrollers" section at the top of "Set up an ESP32"

Reply:
So yes it is very similar, but this is where we get into the question of, do people land on this page first, and does it need to function independently? I definitely think they might, from either docs search, or google etc search, or chatbot, or another page like a tutorial that links here. With it hidden in the expander, what's the harm?

Reply:
Agree that people may land here first. If we move the boards carousel here, this page will actually contain more content (and perhaps more detail) about which platforms we support, and the section in "Learn the basics" is more of a light intro. In that case, can we keep the content here but move it out of the expander?

The reason I'm suggesting this is it feels like important information that should be prominent (not behind an expander) somewhere, and I think I agree this is the place for putting the more detailed version.

Reply:
per offline convo for transparency: decided to combine the supported section expander and the prereq expander on the viam-server install page, and leave this as-is